tap_copper/streams/abstracts.py

```diff
@@ -265,17 +304,25 @@ def get_bookmark(self, state: Dict, stream: str, key: Any = None) -> int:
         return min_parent_bookmark
 
     def write_bookmark(
-        self, state: Dict, stream: str, key: Any = None, value: Any = None
+        self,
+        state: Dict,
+        stream: str,
+        key: Any = None,
+        value: Any = None,
     ) -> Dict:
-        """A wrapper for singer.get_bookmark to deal with compatibility for
-        bookmark values or start values."""
+        """Write bookmark for parent and propagate to children."""
         if self.is_selected():
-            super().write_bookmark(state, stream, value=value)
+            super().write_bookmark(
+                state=state, stream=stream, key=key, value=value
+            )
```

Comment:
Please find other istances of misaligned indentations and fix those like below.

```suggestion
            super().write_bookmark(
                state=state,
                stream=stream,
                key=key,
                value=value
            )
```

Reply:
Done: switched to super().get_bookmark(state=state, 
                                                                 stream=stream, 
                                                                  key=key,
                                                                  value=value
                                                                 ).